engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/transformer/UserTaskTransformer.java

```diff
@@ -105,6 +107,22 @@ private void transformAssignmentDefinition(
     }
   }
 
+  private void transformTaskSchedule(
+      final UserTask element, final JobWorkerProperties jobWorkerProperties) {
+    final var taskSchedule = element.getSingleExtensionElement(ZeebeTaskSchedule.class);
+    if (taskSchedule == null) {
```

Comment:
:wrench: What about putting a new line between if statements? That way, we will separating each guard condition and conversions, thus making it more readable. Or is it against our code conventions?

Reply:
Sure, I'll add them. We don't have any convention on that.

However, I would add the new line before the variable assignments since the variables are used in the if statements.

Reply:
That was exactly what I mean. Thanks!